integration-tests/integration/search/prisonerSearch.cy.js

```diff
@@ -80,7 +80,7 @@ context('Prisoner search', () => {
           data: [inmate1, inmate2],
         })
         cy.task('stubGetIepSummaryForBookingIds', [inmate1Iep, inmate2Iep])
-        cy.visit(`/prisoner-search`)
+        cy.visit(`/prisoner-search?feature=legacy`)
 
```

Comment:
for the cypress test force the testing of both feature routes - this will be remove once this is live